File: lista1_py/3.horario_h_min.py
#3. Leia um valor em minutos, calcule e escreva o equivalente em horas e minutos.

#Entrada
minutos = int(input('Informe o horário em minutos: '))

#Processamento
horas = minutos // 60
min = minutos % 60

#Saída
print(f'\n>>> O horário de {minutos}min equivale a {horas}h e {min}min.')

File: lista1_js/3.horario_h_min.js
//3. Leia um valor em minutos, calcule e escreva o equivalente em horas e minutos.

import { question } from "readline-sync"

//Entrada
const minutos = Number(question('Informe o horário em minutos: '))

//Processamento
const horas = Math.floor(minutos / 60)
const min = minutos % 60

//Saída
console.log(`\n>>> O horário de ${minutos}min equivale a ${horas}h e ${min}min.`)